shared/src/commonTest/kotlin/com/mbta/tid/mbta_app/model/RouteCardDataLeafTest.kt

```diff
@@ -164,6 +164,116 @@ class RouteCardDataLeafTest {
         )
     }
 
+    @Test
+    fun `formats CR as prediction in past with status`() {
```

Comment:
question: I don't know enough about CR statuses - does this happen frequently? Wonder if it would result in most-all CR departures remaining in the tiles for a while after the train has actually departed